distribution/resources/conf/micro-gw-default.conf

```diff
@@ -1,30 +1,55 @@
+# Transport listener Configurations
 [listenerConfig]
-  host="0.0.0.0"
-  httpPort=9090
-  httpsPort=9095
-  keyStorePath="${ballerina.home}/bre/security/ballerinaKeystore.p12"
-  keyStorePassword="ballerina"
-  trustStorePath="${ballerina.home}/bre/security/ballerinaTruststore.p12"
-  trustStorePassword="ballerina"
-  tokenListenerPort=9096
-
+  # Microgateway exposed IP / Host
+  host = "0.0.0.0"
+  # HTTP port that is used to make APIs available to the outside.
+  httpPort = 9090
+  # HTTPS port that is used to make APIs available to the outside and for endpoints(/token, /authorize, /revoke/, userinfo) of Key Manager.
+  httpsPort = 9095
+  # HTTP port for endpoints(/token, /authorize, /revoke/, userinfo) of Key Manager
```

Comment:
HTTPs port

Reply:
Thanks. Fixed in https://github.com/wso2/product-microgateway/pull/953/commits/3c15c88053510495621ea037d2c045a7c6174314